README.md

```diff
@@ -0,0 +1,583 @@
+[![jazzband][jazzband-image]][jazzband]
+[![pypi][pypi-image]][pypi]
+[![pyversions][pyversions-image]][pyversions]
+[![pre-commit][pre-commit-image]][pre-commit]
+[![buildstatus-gha][buildstatus-gha-image]][buildstatus-gha]
+[![codecov][codecov-image]][codecov]
+
+# pip-tools = pip-compile + pip-sync
+
+A set of command line tools to help you keep your `pip`-based packages fresh,
+even when you've pinned them. You do pin them, right? (In building your Python application and its dependencies for production, you want to make sure that your builds are predictable and deterministic.)
+
+[![pip-tools overview for phase II][pip-tools-overview]][pip-tools-overview]
+
+## Installation
+
+Similar to `pip`, `pip-tools` must be installed in each of your project's
+[virtual environments](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments):
+
+```console
```

Comment:
Oh, I see that this isn't highlighted at all.. I wonder if it's a MyST bug and should be reported to @chrisjsewell. Also, could it be because of the inclusion of MD-into-RST and not MD-into-MD? I wonder if we converted `docs/index.rst` to `docs/index.md` first, would it break the same way (with `README.rst` remaining the same)?

Reply:
Yes, #1874 seems broken....